Wiki.py

```diff
@@ -0,0 +1,44 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+def wikiInfo():
+    cat = wikipedia.page("Cat")
```

Comment:
Documentations, such as docstrings and in-line comments, would be very helpful for readers to understand your code. For example, I don't really have a good idea of what `wikiInfo()` function does here. Docstring that describes what this function does would be helpful for me to understand how your code works.